README.md

```diff
@@ -268,46 +275,45 @@ Search for:
 `bottlerocket-update-operator.yaml` pulls operator images from Amazon ECR Public.
 You may also choose to pull from regional Amazon ECR repositories such as the following.
 
-  - 917644944286.dkr.ecr.af-south-1.amazonaws.com
-  - 375569722642.dkr.ecr.ap-east-1.amazonaws.com
-  - 328549459982.dkr.ecr.ap-northeast-1.amazonaws.com
-  - 328549459982.dkr.ecr.ap-northeast-2.amazonaws.com
-  - 328549459982.dkr.ecr.ap-south-1.amazonaws.com
-  - 328549459982.dkr.ecr.ap-southeast-1.amazonaws.com
-  - 328549459982.dkr.ecr.ap-southeast-2.amazonaws.com
-  - 328549459982.dkr.ecr.ca-central-1.amazonaws.com
-  - 328549459982.dkr.ecr.eu-central-1.amazonaws.com
-  - 328549459982.dkr.ecr.eu-north-1.amazonaws.com
-  - 586180183710.dkr.ecr.eu-south-1.amazonaws.com
-  - 328549459982.dkr.ecr.eu-west-1.amazonaws.com
-  - 328549459982.dkr.ecr.eu-west-2.amazonaws.com
-  - 328549459982.dkr.ecr.eu-west-3.amazonaws.com
-  - 509306038620.dkr.ecr.me-south-1.amazonaws.com
-  - 328549459982.dkr.ecr.sa-east-1.amazonaws.com
-  - 328549459982.dkr.ecr.us-east-1.amazonaws.com
-  - 328549459982.dkr.ecr.us-east-2.amazonaws.com
-  - 328549459982.dkr.ecr.us-west-1.amazonaws.com
-  - 328549459982.dkr.ecr.us-west-2.amazonaws.com
-  - 388230364387.dkr.ecr.us-gov-east-1.amazonaws.com
-  - 347163068887.dkr.ecr.us-gov-west-1.amazonaws.com
+  - `917644944286.dkr.ecr.af-south-1.amazonaws.com`
+  - `375569722642.dkr.ecr.ap-east-1.amazonaws.com`
+  - `328549459982.dkr.ecr.ap-northeast-1.amazonaws.com`
+  - `328549459982.dkr.ecr.ap-northeast-2.amazonaws.com`
+  - `328549459982.dkr.ecr.ap-south-1.amazonaws.com`
+  - `328549459982.dkr.ecr.ap-southeast-1.amazonaws.com`
+  - `328549459982.dkr.ecr.ap-southeast-2.amazonaws.com`
+  - `328549459982.dkr.ecr.ca-central-1.amazonaws.com`
+  - `328549459982.dkr.ecr.eu-central-1.amazonaws.com`
+  - `328549459982.dkr.ecr.eu-north-1.amazonaws.com`
+  - `586180183710.dkr.ecr.eu-south-1.amazonaws.com`
+  - `328549459982.dkr.ecr.eu-west-1.amazonaws.com`
+  - `328549459982.dkr.ecr.eu-west-2.amazonaws.com`
+  - `328549459982.dkr.ecr.eu-west-3.amazonaws.com`
+  - `509306038620.dkr.ecr.me-south-1.amazonaws.com`
+  - `328549459982.dkr.ecr.sa-east-1.amazonaws.com`
+  - `328549459982.dkr.ecr.us-east-1.amazonaws.com`
+  - `328549459982.dkr.ecr.us-east-2.amazonaws.com`
+  - `328549459982.dkr.ecr.us-west-1.amazonaws.com`
+  - `328549459982.dkr.ecr.us-west-2.amazonaws.com`
+  - `388230364387.dkr.ecr.us-gov-east-1.amazonaws.com`
+  - `347163068887.dkr.ecr.us-gov-west-1.amazonaws.com`
+  - `183470599484.dkr.ecr.cn-north-1.amazonaws.com.cn`
+  - `183901325759.dkr.ecr.cn-northwest-1.amazonaws.com.cn`
 
```

Comment:
we miss `328549459982.dkr.ecr.ap-northeast-3.amazonaws.com`

Reply:
I think you also need to add:

```
386774335080.dkr.ecr.ap-southeast-3.amazonaws.com
553577323255.dkr.ecr.me-central-1.amazonaws.com
```